controllers/sriovnetwork_controller_test.go

```diff
@@ -213,3 +216,34 @@ var _ = Describe("SriovNetwork Controller", func() {
 		})
 	})
 })
+
+func generateExpectedNetConfig(cr *sriovnetworkv1.SriovNetwork) string {
```

Comment:
Moving these methods around is not something that is directly related to this PR so im not a fan of mixing the two.
if they are mixed then at least it should be in separate commits.


also im not too sure if these methods are used externally from /test/util.go
@fedepaol @SchSeba